### PR TITLE
Update Microsoft.xml

### PR DESCRIPTION
**Pull request by gloomy-ghost**
Add sysdev.microsoft.com

### CODE REVIEW

File: src/chrome/content/rules/Microsoft.xml [hunk 1]
```diff
@@ -88,60 +86,91 @@
...
+			- editorialentitytracking.si.bingads ᴿ
+			- irt.si.bingads               ʳ
+			- reporting.si.bingads         ᴹ
+			- secure.si.bingads            ᴹ
+			- express.tip.bingads          ᴹ
+			- innovation.connect  (redirects to connect.microsoft.com)
+			- services.connect	  ᴹ
 			- silverlight.dlservice *
-			- web.esd *		(esd, origin, origin-web, and web don't exist)
-			- events		(Mismatched)
```
Comment (J0WI) [inline src/chrome/content/rules/Microsoft.xml]: events.microsoft.com WFM

File: src/chrome/content/rules/Microsoft.xml [hunk 2]
```diff
@@ -286,11 +298,31 @@
...
 	<target host="assets.microsoft.com" />
 	<target host="azure.microsoft.com" />
+	
 	<target host="bingads.microsoft.com" />
-
+	<target host="adinquiry.bingads.microsoft.com" />
+	<target host="ads.bingads.microsoft.com" />
+	<target host="adslabs.bingads.microsoft.com" />
 	<target host="advertise.bingads.microsoft.com" />
-	<target host="resources.bingads.microsoft.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Microsoft.xml]: Why has this target been removed?

Comment (gloomy-ghost) [inline src/chrome/content/rules/Microsoft.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/172861773#L396

Good catch, I was removing resources.bc.bingads.microsoft.com actually...

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**